bin/stacks/routing-caching-stack.ts

```diff
@@ -235,7 +235,7 @@ export class RoutingCachingStack extends cdk.NestedStack {
             period: Duration.minutes(60),
             statistic: 'sum',
           }),
-          threshold: 9,
+          threshold: 13,
```

Comment:
Do you know roughly how many times this is failing in the 60 min period currently? Would making this 13 be a high enough threshold?

Reply:
right now it is failing at most 11 times in an hour. 13 should not be hit unless something is really wrong